### PR TITLE
ConfigCheck for aggregators

### PR DESCRIPTION
**Pull request by pepov**
#1746

### CODE REVIEW

File: pkg/sdk/logging/api/v1beta1/fluentd_types.go [hunk 1]
```diff
@@ -113,6 +113,9 @@ type FluentdSpec struct {
 	// Available in Logging operator version 4.5 and later.
 	// Configure sidecar container in Fluentd pods, for example: [https://github.com/kube-logging/logging-operator/config/samples/logging_logging_fluentd_sidecars.yaml](https://github.com/kube-logging/logging-operator/config/samples/logging_logging_fluentd_sidecars.yaml).
 	SidecarContainers []corev1.Container `json:"sidecarContainers,omitempty"`
+	// Overrides the default logging level configCheck setup
```
Comment (OverOrion) [inline pkg/sdk/logging/api/v1beta1/fluentd_types.go:116]:
```suggestion
	// Overrides the default logging level configCheck setup.
```

File: pkg/sdk/logging/api/v1beta1/fluentd_types.go [hunk 2]
```diff
@@ -113,6 +113,9 @@ type FluentdSpec struct {
...
+	// This field is not used directly, just copied over the field in the logging resource if defined
```
Comment (OverOrion) [inline pkg/sdk/logging/api/v1beta1/fluentd_types.go:117]:
```suggestion
	// This field is not used directly, just copied over the field in the logging resource if defined.
```

File: pkg/sdk/logging/api/v1beta1/logging_types.go [hunk 3]
```diff
@@ -47,7 +47,8 @@ type LoggingSpec struct {
 	SkipInvalidResources bool `json:"skipInvalidResources,omitempty"`
 	// Override generated config. This is a *raw* configuration string for troubleshooting purposes.
 	FlowConfigOverride string `json:"flowConfigOverride,omitempty"`
-	// ConfigCheck settings that apply to both fluentd and syslog-ng
+	// ConfigCheck settings that apply to both fluentd or syslog-ng
+	// Can be overridden on the fluentd / syslog-ng level
```
Comment (OverOrion) [inline pkg/sdk/logging/api/v1beta1/logging_types.go:51]:
```suggestion
	// ConfigCheck settings that apply to both fluentd or syslog-ng.
	// Can be overridden on the fluentd / syslog-ng level.
```

File: pkg/sdk/logging/api/v1beta1/syslogng_types.go [hunk 4]
```diff
@@ -58,7 +58,9 @@ type SyslogNGSpec struct {
 	// Available in Logging operator version 4.5 and later.
 	// Create [custom log metrics for sources and outputs]({{< relref "/docs/examples/custom-syslog-ng-metrics.md" >}}).
 	SourceMetrics []filter.MetricsProbe `json:"sourceMetrics,omitempty"`
-	// TODO: option to turn on/off buffer volume PVC
+	// Overrides the default logging level configCheck setup
+	// This field is not used directly, just copied over the field in the logging resource if defined
```
Comment (OverOrion) [inline pkg/sdk/logging/api/v1beta1/syslogng_types.go:62]:
```suggestion
	// Overrides the default logging level configCheck setup.
	// This field is not used directly, just copied over the field in the logging resource if defined.
```

**Review by OverOrion — APPROVED** [review]
LGTM, the comments need `.` so the generated docs look nice.

**Review by kristofgyuracz — APPROVED** [review]
I agree with the suggested changes.

**Pull request merged**